pygen/pygen_src/riscv_instr_cover_group.py

```diff
@@ -2037,6 +2036,5419 @@ def __init__(self):
                                                   "Aligned"    : vsc.bin(1),
                                                   "Not-aligned": vsc.bin(0)
                                                    })
+    '''Floating instructions'''
+
+    @vsc.covergroup
+    class flw_cg(object):
+        def __init__(self, precision, ignore):
+            super().__init__()
+
+            self.instr = None
+            self.options.ignore = ignore
+            self.cp_rs1 = vsc.coverpoint(lambda: self.instr.rs1,
+                                         cp_t=vsc.enum_t(riscv_reg_ex_zero_t))
+            self.cp_fd = vsc.coverpoint(lambda: self.instr.fd,
+                                        cp_t=vsc.enum_t(riscv_fpr_t))
+            self.cp_imm_sign = vsc.coverpoint(lambda: self.instr.imm_sign,
+                                              cp_t=vsc.enum_t(operand_sign_e))
+            self.cp_gpr_hazard = vsc.coverpoint(lambda: self.instr.gpr_hazard,
+                                                cp_t=vsc.enum_t(hazard_e))
+            self.cp_lsu_hazard = vsc.coverpoint(lambda: self.instr.lsu_hazard,
+                                                cp_t=vsc.enum_t(
+                                                    branch_hazard_e))
+            self.cp_sfp_special_values_on_fd_value = vsc.coverpoint(lambda:
+                                                                    self.instr.fd_value[31:0],
+                                                                    bins={"infinity":
+                                                                          vsc.bin_array(
+                                                                              [], 2139095040,

```

Comment:
Can we define these constants as parameters, then use the parameter name (or "parameter_name -1") instead?